airbyte-integrations/bases/source-acceptance-tests/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM openjdk:14.0.2-slim
+
+# Install Docker to launch worker images. Eventually should be replaced with Docker-java.
+# See https://gitter.im/docker-java/docker-java?at=5f3eb87ba8c1780176603f4e for more information on why we are not currently using Docker-java
+RUN apt-get update && apt-get install -y \
+                          apt-transport-https \
+                          ca-certificates \
+                          curl \
+                          gnupg-agent \
+                          software-properties-common
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
+RUN add-apt-repository \
+       "deb [arch=amd64] https://download.docker.com/linux/debian \
+       $(lsb_release -cs) \
+       stable"
+RUN apt-get update && apt-get install -y docker-ce-cli jq
+
+ENV APPLICATION source-acceptance-tests
+
+WORKDIR /app
+
+COPY entrypoint.sh .
+COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
+
+RUN tar xf ${APPLICATION}.tar --strip-components=1
+
+ENTRYPOINT ["/app/entrypoint.sh"]
+
+LABEL io.airbyte.version=0.1.0
```

Comment:
note that we will now publish this just like any other connector so people can use this completely from Docker without needing any build steps